prime-angular-frontend/src/app/modules/phsa-eforms/pages/available-access/available-access.component.ts

```diff
@@ -44,7 +47,9 @@ export class AvailableAccessComponent implements OnInit {
   public onSubmit(): void {
     if (this.formUtilsService.checkValidity(this.form)) {
       const payload = this.phsaEformsFormStateService.json;
-      this.busy = this.phsaEformsResource.createEnrollee(payload)
+      this.busy = of(noop).pipe(
```

Comment:
`noop` is a method with a void return type producing undefined so you should be using `noop()` which was in the example I added in the last comment.

Reply:
Wops, have to change its use somewhere else too!